test_dynamic_search.py

```
@@ -0,0 +1,34 @@
+
+class TestSearchParser(unittest.TestCase):
+    def test_single_field(self):
+        self.assertEqual(
+            parse_search_query("char:wolverine"),
+            'characters: "wolverine"'
+        )
+
+    def test_multiple_fields(self):
+        self.assertEqual(
+            parse_search_query("#mutant &marvel char:logan"),
+            'tags: "mutant" AND publisher: "marvel" AND characters: "logan"'
```

Ruff (0.12.2)

Trailing comma missing

Add trailing comma

(COM812)